go-controller/pkg/cni/cni.go

```diff
@@ -136,27 +135,22 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, podLister corev1listers.PodL
 		response.PodIFInfo = podInterfaceInfo
 	}
 
-	responseBytes, err := json.Marshal(response)
-	if err != nil {
-		return nil, fmt.Errorf("failed to marshal pod request response: %v", err)
-	}
-
-	return responseBytes, nil
+	return response, nil
 }
 
-func (pr *PodRequest) cmdDel() ([]byte, error) {
+func (pr *PodRequest) cmdDel() (*Response, error) {
```

Comment:
`cmdDel` and `cmdCheck` never return any actual response data. You can just remove the first argument from them

Reply:
done!